src/no-op-provider.ts

```diff
@@ -0,0 +1,39 @@
+import { Provider, ResolutionDetails } from './types';
+
+const REASON_NO_OP = 'No-op';
+
+/**
+ * The No-op provider is set by default, and simply always returns the default value.
+ */
+class NoopFeatureProvider implements Provider {
+  readonly name = 'No-op Provider';
+
+  isEnabled(_: string, defaultValue: boolean): Promise<boolean> {
```

Comment:
I thought we got rid of `isEnabled`

Reply:
We did, I will remove this. I auto implemented the interface, and at the time I did that is had this method.